apps/store/src/components/Header/HeaderStyles.tsx

```diff
@@ -105,12 +107,13 @@ export const NavigationPrimaryList = styled(NavigationMenuPrimitive.List)({
     flexDirection: 'row',
     alignItems: 'center',
     height: MENU_BAR_HEIGHT_DESKTOP,
+    padding: 0,
```

Comment:
nit + question:
Should we be using the following instead? 🤔 
 
```suggestion
    padding: theme.space.none,
```

Reply:
I didn't know we had that one, will update!